src/PiecewiseLinearOpt.jl

```diff
@@ -3,8 +3,7 @@ __precompile__()
 module PiecewiseLinearOpt
 
 import JuMP
-import MathOptInterface
-const MOI = MathOptInterface
+const MOI = JuMP.MOI
```

Comment:
We should add a direct dependency on MathOptInterface and do
```julia
import MathOptInterface as MOI
```
Let's be explicit instead of clever

Reply:
The usage of MathOptInterface was related to functionality that now is directly available in JuMP (e.g. SOS2), so I just removed all MOI references.